.github/workflows/java.yml

```diff
@@ -61,18 +61,22 @@ jobs:
           sudo apt-get install llvm-11 clang-11 lld-11 libclang-11-dev libz-dev -y
           
       - name: Code Format Check
+        working-directory: java
         run: |
           export JAVA_HOME=${JAVA_HOME_11_X64}
-          pushd java 
           mvn spotless:check
```

Comment:
Maybe let's add --no-transfer-progress in  java too? Without that option, it will be much harder to read logs in GitHub actions. I would recommend using that option in every place in CI; that option was introduced for CI builds mostly to make logs readable.

Reply:
Done